heron/api/src/java/org/apache/heron/api/metric/LatencyStatAndMetric.java

```diff
@@ -225,7 +225,7 @@ private synchronized void rotate(long lat, long count, long timeSpent, long targ
   }
 
   synchronized Map<String, Double> getTimeLatAvg(long now) {
-    Map<String, Double> ret = new HashMap<>();
+    Map<String, Double> ret = new ConcurrentHashMap<>();
```

Comment:
ConcurrentHashMap  is not needed here I think. This is a get function and it is already synchronized.